app/templates/users/details.html

```diff
@@ -12,8 +18,50 @@
 
 {% block main_column %}
 
+{% if current_user.is_superuser %}
+<form action="{{ url_for('users.edit', { id: user.auth0_id | urlencode }) }}" method="post">
+
+  {% call fieldset(
+    "superadmin",
+    "Super Admin",
+    hint="User is a super admin, allowing all privileges on the Control panel."
+  ) -%}
+    <div class="govuk-checkboxes">
+      {{ checkbox_field("superadmin", "Super Admin", user.is_superuser) }}
+    </div>
+  {%- endcall %}
+
+  <div class="form-group">
+    <input type="submit" class="button button-secondary" value="Save changes">
+  </div>
+
+</form>
+
+<form action="{{ url_for('users.reset_mfa', { id: user.auth0_id | urlencode }) }}" method="post">
+
+  {% call fieldset(
+    "resetmfa",
+    "Reset MFA",
+    hint="Reset the user's multi-factor authentication settings, forcing them to reconfigure."
+  ) -%}
+    <div class="form-group">
+      <button class="button button-secondary">Reset MFA</button>
+    </div>
+  {%- endcall %}
+
+</form>
+{% endif %}
 
 {% include 'users/includes/tables.html' %}
 
+{% if current_user.is_superuser %}
+<hr>
+
+<form action="{{ url_for("users.delete", { id: user.auth0_id }) }}" method="post">
+
+  {{ delete_button("Delete user", confirm="Are you sure you want to delete this user?") }}
```

Comment:
I can't remember how we handle this in the backend (CP-API), what are the implications of deleting a user? Is it just "delete from DB" or we do more, e.g. uninstall tools helm charts, uninstall init/config helmcharts, namespace, etc...?

Reply:
The delete user button already exists in the user list page. I just put a copy on the user details page as well. Changing what it means to delete a user is not in scope of this change and would be in the control panel api anyway.